src/amuse/units/amuse_2010/astronomical_constants.py

```diff
@@ -0,0 +1,19 @@
+"""
+Series of astronomical constants, which are in turn used by the units of the
+same names.
+"""
+
+import numpy as np
+from amuse.units.si import m, kg
+from amuse.units.derivedsi import W, km
+
+
+au = 149597870691.0 | m
+parsec = au / np.tan(np.pi / (180 * 60 * 60))
```

Comment:
Note that both 'au' and 'parsec' are also defined as units... Be careful, this may clash!
For Lsun etc below, note that here they only have one capital letter, the unit has LSun etc... This is to be more in line with Astropy etc, which only have one capital letter. Still, it may be better to be consistent within AMUSE?

Reply:
normally it wouldn't be a problem, since units would be used as `units.(unitname)` rather than being directly imported. But I don't exactly like it.

Reply:
Are constants that can also be units actually ever used as anything else than units?

Reply:
Probably not. Except for making the units.

Reply:
So... We could in principle make the names here (way) more descriptive, e.g.:
```
distance_parsec = ...
distance_au = ...
luminosity_sun = ...
radius_sun = ...
```
That would solve this issue. It's very verbose but since you're always using the unit form that should not be a problem.

Reply:
Well, they could still have the same names actually, as long as no one directly uses the `amuse.units.amuse_2010.astronomical_constants` module. It should only be used in the modules that define the units then, and we could even name it `_astronomical_constants` to mark it as private.

Then in the modules where the units are defined, we could `import amuse.units.amuse_2010.astronomical_constants as ac` and write `ac.au` and the like everywhere to clearly point out where they're coming from and that these are the constants and not the units. I think that's what I'd do.